packages/cli/src/commands/serve.ts

```diff
@@ -6,26 +6,39 @@ import { viteExternalsPlugin } from 'vite-plugin-externals';
 import { Logger } from '../utils/logger.js';
 import { getAppBridgeVersion } from '../utils/appBridgeVersion.js';
 import pkg from '../../package.json';
+import path from 'path';
+import { fileURLToPath } from 'url';
 
 class PlatformAppDevelopmentServer {
     constructor(private readonly port: number) {}
 
     async serve(): Promise<void> {
+        Logger.info('Development');
+
+        const __dirname = path.dirname(fileURLToPath(import.meta.url));
         try {
             const viteServer = await createServer({
                 root: process.cwd(),
-                configFile: false,
-                plugins: [
-                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                    //@ts-ignore
-                    react(),
-                ],
+                plugins: [react()],
                 define: {
                     'process.env.NODE_ENV': JSON.stringify('development'),
                 },
-                base: `http://localhost:${this.port}/`,
+                build: {
+                    lib: {
+                        entry: path.resolve(__dirname, 'src/index.tsx'),
+                        name: 'platformApp',
+                        formats: ['es', 'umd'],
```

Comment:
```suggestion
                        formats: ['es'],
```

Reply:
yeah your right i dont need it anymore. When we imported the module into the index.html i didnt declare it as a module and though ill need umd. Removed it 👍

Reply:
for context, were planning to have smth like this for our index.html

`<script type='module'>
    import platformApp from '/src/index.tsx';
    import ReactDOM from 'react-dom/client';
    import React from 'react';

    console.log('platformApp', platformApp.settings);

    const root = ReactDOM.createRoot(document.getElementById('root'));
    root.render(React.createElement(platformApp.app));
</script>`